ckanext/qgov/common/anti_csrf.py

```diff
@@ -232,11 +236,17 @@ def create_response_token():
     message = "{}/{}/{}".format(timestamp, nonce, username)
     token = "{}!{}".format(get_digest(message), message)
 
-    _set_response_token_cookie(token)
+    # pre-emptively set the token cookie if using Pylons,
+    # otherwise assume the after_app_request hook will do it
+    try:
+        set_response_token_cookie(token, pylons_response)
+    except Exception as e:
+        LOG.warn("Unable to set CSRF token cookie via Pylons: %s", e)
```

Comment:
should this be info not warn. if pylons is not enabled (old ckan) would this not fill the logs?